src/sentry/web/frontend/twofactor.py

```diff
@@ -142,3 +143,18 @@ def handle(self, request):
             'other_interfaces': self.get_other_interfaces(interface, interfaces),
             'activation': activation,
         }, request, status=200)
+
+
+def u2f_appid(request):
```

Comment:
Needs to be public as per https://fidoalliance.org/specs/fido-u2f-v1.0-ps-20141009/fido-appid-and-facets-ps-20141009.html